docs-content/using.html.md.erb

```diff
@@ -72,33 +56,57 @@ The table below summarizes the environment variables, their default and allowed
       <td>false</td>
     </tr>
     <tr>
-      <td>APPD_SINK</td>
-      <td>Sink to which the metrics are to be pushed</td>
-      <td>stdout/Controller</td>
-      <td>Controller</td>
+    <td>APPD_NOZZLE_APP</td>
+    <td>Name of the Nozzle Application under which the metrics are recorded</td>
+    <td>Any string</td>
+    <td>appd-nozzle</td>
+    </tr>
+    <tr>
+    <td>APPD_NOZZLE_TIER</td>
+    <td>Name of the Nozzle Tier under which the metrics are recorded</td>
+    <td>Any string</td>
+    <td>appd-nozzle-tier</td>
+    </tr>
+    <tr>
+    <td>APPD_NOZZLE_Node</td>
+    <td>Name of the Nozzle Node under which the metrics are recorded</td>
+    <td>Any string</td>
+    <td>appd-nozzle-node</td>
     </tr>
     <tr>
       <td>APPD_SAMPLING_RATE</td>
       <td>Polling Interval in secs to Firehose Nozzle</td>
       <td>number of seconds</td>
       <td>2 seconds</td>
     </tr>
+    <tr>
+      <td>APPD_SINK</td>
+      <td>Sink to which the metrics are to be pushed</td>
+      <td>stdout/Controller</td>
+      <td>Controller</td>
+    </tr>
     <tr>
       <td>APPD_SSL_ENABLED</td>
       <td>Enable/Disable SSL to Controller</td>
       <td>true / false</td>
       <td>false</td>
     </tr>
     <tr>
-      <td>NOZZLE_UAA_URL</td>
-      <td>UAA API endpoint URL</td>
-      <td>Run cf curl /v2/info and record the UAA endpoint</td>
+      <td>NOZZLE_PASSWORD</td>
+      <td>Password for the above account</td>
+      <td>none</td>
       <td>none</td>
     </tr>
     <tr>
-      <td>NOZZLE_TRAFFIC_CONTROLLER_URL</td>
-      <td>Doppler end point URL</td>
-      <td>Run cf curl /v2/info and record doppler api url</td>
+    <td>NOZZLE_TRAFFIC_CONTROLLER_URL</td>
```

Comment:
For this Environment variable is says **NOZZLE_TRAFFIC_CONTROLLER_URL**, in the User Interface of the Settings tab, select AppDynamics Nozzle Configuration it asks for a Doppler Logging Endpoint. In the cf command it is also listed as `doppler-logging-endpoint`. 
For this page, and the Installation and Configuring page, I'm unclear as to why we have these two different terms?